retriever/lib/datapackage.py

```diff
@@ -170,16 +170,17 @@ def create_json():
     contents['resources'] = []
     contents['retriever'] = "True"
     contents['retriever_minimum_version'] = "2.0.dev"
-    contents['encoding'] = clean_input("encoding: ", ignore_empty = True)
-    if is_empty(clean_input("encoding: ", ignore_empty = True)) : contents['encoding'] = ENCODING
-    contents['version'] = "1.0.0";
+    contents['encoding'] = clean_input("encoding: ", ignore_empty=True)
+    if is_empty(clean_input("encoding: ", ignore_empty=True)):
+        contents['encoding'] = ENCODING
+        contents['version'] = "1.0.0"
```

Comment:
You have changed the statement and changed the logic too.
Please check this statement again.
Also provide a sample script that you created with this sample code